package.json

```diff
@@ -22,10 +22,10 @@
     "preview": "vite preview",
     "start": "run-p start:server start:watch:ts start:watch:css",
     "start:server": "vite",
-    "start:watch:ts": "tsc --watch",
+    "start:watch:ts": "npm run build:src:js -- --watch",
     "start:watch:css": "chokidar \"./src/**/*.scss\" -c \"npm run build:src:css\"",
     "build:pkg": "node scripts/package-json.js",
-    "build:src:js": "tsc -p tsconfig.json",
+    "build:src:js": "tsc -p tsconfig.json --inlineSources --sourceMap",
```

Comment:
Source maps are required to resolve test coverage for imports from `lib/components/` path. Also, nice dev experience improvement for our consumers